### PR TITLE
Add docs for cloud development

### PR DESCRIPTION
**Pull request by sanjaysiddhanti**
@di @brainwane 

This PR documents my setup for developing Warehouse in the cloud.

My laptop is too slow to reliably run Warehouse locally, so I run it in the cloud, which works seamlessly. Dustin mentioned it could be worth documenting for others to use.

Let me know if you have feedback or if you think I should link to this from the "Getting started" docs.

### CODE REVIEW

**Comment by sanjaysiddhanti** [comment]
Sure. I pushed an update.

File: docs/development/cloud.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+and costs can be saved by terminating the instance if you don't plan to use it
+for a few days.
+
+Development setup
+-----------------
+
+SSH into the cloud server. Create a new SSH key on this server and associate
+it with your Github account. Clone the Warehouse repo and go through the
+normal setup instructions for building the Docker images and starting the
+containers.
```
Comment (eamanu) [inline docs/development/cloud.rst:27]: Maybe could be great have  a link to ```https://warehouse.pypa.io/development/getting-started/?highlight=Makefile#configure-the-development-environment```

Comment (sanjaysiddhanti) [reply docs/development/cloud.rst:27]: 5eb398a updated, thanks for the suggestion

File: docs/development/cloud.rst [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+on the cloud provider, instances are available for approximately $20 / month.
+Costs can be saved by terminating the instance if you don't plan to use it
+for a few days.
+
+Development setup
+-----------------
+
+SSH into the cloud server. Create a new SSH key on this server and associate
+it with your Github account. Clone the Warehouse repo and go through the
+:ref:`normal setup instructions <running-warehouse-containers>` for building the Docker images and starting the
```
Comment (eamanu) [inline docs/development/cloud.rst:26]: please, take care here on the length of the line

File: docs/development/cloud.rst (hunk 1 above)
Comment (eamanu) [reply docs/development/cloud.rst:27]: Great

File: docs/development/cloud.rst [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+containers.
```
Comment (di) [inline docs/development/cloud.rst:27]:
```suggestion
:ref:`normal setup instructions <running-warehouse-containers>` for building
the Docker images and starting the containers.
```

**Comment by sanjaysiddhanti** [comment]
Thanks @di !

**Pull request merged**